lib/logic/entities/Entity.js

```diff
@@ -0,0 +1,45 @@
+const yaml   = require('js-yaml');
+
+class Entity {
+    toDefault() {
+        return this.extractValues('default');
+    }
+    toWide() {
+        return this.extractValues('wide');
+    }
+
+    toJson() {
+        return JSON.stringify(this.info, null , '\t');
+    }
+
+    toYaml() {
+        return yaml.safeDump(this.info);
+    }
+
+    toName() {
+        return this.entityType + '/' + this.info.name;
+    }
+
+    extractValues(type) {
```

Comment:
please pass here the actual array that you want to go over.
no point in passing the type again